recipes/cvplot/all/conanfile.py

```diff
@@ -0,0 +1,33 @@
+from conans import ConanFile, tools
+import os
+
+required_conan_version = ">=1.33.0"
+
+
+class CvPlotConan(ConanFile):
+    name = "cvplot"
+    description = "fast modular opencv plotting library"
+    license = "MIT License"
```

Comment:
```suggestion
    license = "MIT"
```

Use the short name: https://spdx.org/licenses/MIT